react/features/filmstrip/functions.any.ts

```diff
@@ -42,12 +42,19 @@ export function updateRemoteParticipants(store: IStore, force?: boolean, partici
     const sharedVideos = fakeParticipants ? Array.from(fakeParticipants.keys()) : [];
     const speakers = new Map();
     const dominant = dominantSpeaker ? getParticipantById(state, dominantSpeaker) : undefined;
+    const config = state['features/base/config'];
+    const defaultRemoteDisplayName = config?.defaultRemoteDisplayName ?? 'Fellow Jitster';
 
     // Generate the remote active speakers list.
     if (dominant && !dominant.local) {
         speakers.set(dominant.id, dominant.name);
     }
-    previousSpeakers.forEach((name, id) => speakers.set(id, name));
+    previousSpeakers.forEach(id => {
+        const participant = getParticipantById(state, id);
+        const displayName = participant?.name ?? defaultRemoteDisplayName;
+
+        speakers.set(id, displayName);
```

Comment:
Why do we still need this map? 

Can't we get the display name as part of the sort method and compare and return? This would save us the reduce call after this. WDYT?

Reply:
Makes sense.